api/fixtures/example.go

```diff
@@ -61,18 +61,16 @@ type ExampleOptions struct {
 }
 
 type ExampleAWSOptions struct {
-	Region                                 string
-	Zone                                   string
-	VPCID                                  string
-	SubnetID                               string
-	SecurityGroupID                        string
-	InstanceProfile                        string
-	InstanceType                           string
-	Roles                                  []hyperv1.AWSRoleCredentials
-	KubeCloudControllerUserAccessKeyID     string
-	KubeCloudControllerUserAccessKeySecret string
-	NodePoolManagementUserAccessKeyID      string
-	NodePoolManagementUserAccessKeySecret  string
+	Region                     string
+	Zone                       string
+	VPCID                      string
+	SubnetID                   string
+	SecurityGroupID            string
+	InstanceProfile            string
+	InstanceType               string
+	Roles                      []hyperv1.AWSRoleCredentials
+	KubeCloudControllerRoleARN string
```

Comment:
why do we need to explicitly call this now? can't it be just another Roles item now, like openshift-ingress?

Reply:
it is plumbed in a different way since `roles` is destined for components in the guest cluster, not the management cluster